src/main/java/internet/shop/controller/product/AccessAllProductsController.java

```diff
@@ -0,0 +1,26 @@
+package internet.shop.controller.product;
+
+import internet.shop.lib.Injector;
+import internet.shop.model.Product;
+import internet.shop.service.ProductService;
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class AccessAllProductsController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("internet.shop");
+    private ProductService productService
+            = (ProductService) injector.getInstance(ProductService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        List<Product> allProducts = productService.getAll();
+
+        req.setAttribute("products", allProducts);
+        req.getRequestDispatcher("/WEB-INF/views/products/delete.jsp").forward(req, resp);
```

Comment:
```suggestion
        req.getRequestDispatcher("/WEB-INF/views/products/manage.jsp").forward(req, resp);
```

let's create an informative name for this jsp